bindings/lua/Cargo.toml

```diff
@@ -29,9 +29,31 @@ rust-version.workspace = true
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+
+[features]
+default = ["mlua/lua52"]
+lua52 = ["mlua", "mlua/lua52"]
+
+# todo: fix mlua multi-version in `cargo build --all-features` error
```

Comment:
We can submit a new issue for this and remove those commented code.

Reply:
Ok